synchronizer/l1_workers_decorator_limit_retries_by_time.go

```diff
@@ -0,0 +1,70 @@
+package synchronizer
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/0xPolygonHermez/zkevm-node/log"
+)
+
+const (
+	cleanUpOlderThan = time.Hour
+)
+
+type controlWorkerFlux struct {
+	time    time.Time
+	retries int
+}
+
+type workerDecoratorLimitRetriesByTime struct {
+	*workers
+	processingRanges    liveBlockRangesGeneric[controlWorkerFlux]
```

Comment:
I think this controlWorkerFlux shouldn't use liveBlockRanges. This is a completely different structure that is not related with blockRanges.

Reply:
That is the reason of using `tag`